docs/cryptodoc/src/10_tpm.rst

```diff
@@ -65,6 +65,7 @@ capabilities and fundamental key management operations.
    - If no handle was provided, ``handle = next_free_handle()`` (via ``Esys_GetCapability``)
    - ``Esys_EvictControl(ctx, key.transient_handle(), sessions, out(handle))``
    - ``Esys_TR_SetAuth(ctx, key.transient_handle(), auth_value)``
+   - ``key.persistent_handle() = Esys_TR_GetTpmHandle(ctx, key.transient_handle())``
```

Comment:
Huh? It is `m_impl->m_ctx` which is the same as for the other ESAPI invocations.

Reply:
Yes, but this has been abstracted to `m_esys_ctx` in this documentation (there's no way to see what `ctx`is based on the Input description of the method documented here.
In the other ESAPI invocations, ctx has been listed as an input (and for the docu I'm fine with not distinguishing between the wrapped and original context)

Reply:
Fair point. I'll just add the `ctx` as an input value. Technically, this would be the `this` pointer here, I believe that's why I left it out of the inputs. But that might not be obvious at all.

I think, we should definitely abstract from the context's inner handles here.